### PR TITLE
Minor fixes of Set document

### PR DESCRIPTION
**Pull request by myshov**
### Description

Minor fixes in the comments of the code and in the main text of the document

### Motivation

Wanted to make the page more consistent

### Related issues and pull requests

Related to https://github.com/mdn/translated-content/pull/11473

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Hi, we deliberately use the lower-case, non-code form to refer to instances where possible, because it's closer to colloquial language and makes the text less choppy. I like the other changes, though.

**Comment by myshov** [comment]
@Josh-Cena hi! Thanks for review! Fixed 👌

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The **`Set`** object lets you store unique values of any type, whether {{Glossar
 
 ## Description
 
-`Set` objects are collections of values. A value in the `Set` **may only occur once**; it is unique in the `Set`'s collection. You can iterate through the elements of a set in insertion order. The _insertion order_ corresponds to the order in which each element was inserted into the set by the [`add()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add) method successfully (that is, there wasn't an identical element already in the set when `add()` was called).
+Set objects are collections of values. A value in the set **may only occur once**; it is unique in the set's collection. You can iterate through the elements of a set in insertion order. The _insertion order_ corresponds to the order in which each element was inserted into the set by the [`add()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add) method successfully (that is, there wasn't an identical element already in the set when `add()` was called).
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:14]:
```suggestion
`Set` objects are collections of values. A value in the set **may only occur once**; it is unique in the set's collection. You can iterate through the elements of a set in insertion order. The _insertion order_ corresponds to the order in which each element was inserted into the set by the [`add()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add) method successfully (that is, there wasn't an identical element already in the set when `add()` was called).
```

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Value equality is based on the [SameValueZero](/en-US/docs/Web/JavaScript/Equali
 
 ### Performance
 
-The `Set` [`has`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/has) method checks if a value is in a `Set` object, using an approach that is, on average, quicker than testing most of the elements that have previously been added to the `Set` object. In particular, it is, on average, faster than the [`Array.prototype.includes`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method when an `Array` object has a `length` equal to a `Set` object's `size`.
+The set's [`has`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/has) method checks if a value is in a set object, using an approach that is, on average, quicker than testing most of the elements that have previously been added to the set object. In particular, it is, on average, faster than the [`Array.prototype.includes`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method when an array object has a `length` equal to a set object's `size`.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:24]:
```suggestion
The [`has`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/has) method checks if a value is in the set, using an approach that is, on average, quicker than testing most of the elements that have previously been added to the set. In particular, it is, on average, faster than the [`Array.prototype.includes`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method when an array has a `length` equal to a set's `size`.
```

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ These properties are defined on `Set.prototype` and shared by all `Set` instance
 
 ## Examples
 
-### Using the Set object
+### Using the set object
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:74]:
```suggestion
### Using the Set object
```

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 4]
```diff
@@ -148,7 +148,7 @@ const intersection = new Set([...mySet1].filter((x) => mySet2.has(x)));
 // difference can be simulated via
 const difference = new Set([...mySet1].filter((x) => !mySet2.has(x)));
 
-// Iterate set entries with forEach()
+// Iterate set's entries with forEach()
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:151]:
```suggestion
// Iterate set entries with forEach()
```

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 5]
```diff
@@ -220,7 +220,7 @@ symmetricDifference(setA, setC); // returns Set {1, 2, 5, 6}
 difference(setA, setC); // returns Set {1, 2}
 ```
 
-### Relation with Array objects
+### Relation to array objects
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:223]:
```suggestion
### Relation to arrays
```

File: files/en-us/web/javascript/reference/global_objects/set/index.md [hunk 6]
```diff
@@ -246,20 +246,20 @@ console.log([...new Set(numbers)]);
...
 mySet.size; // 5
 
-//case sensitive & duplicate omission
+// case sensitive & duplicate omission
 new Set("Firefox"); // Set(7) { "F", "i", "r", "e", "f", "o", "x" }
 new Set("firefox"); // Set(6) { "f", "i", "r", "e", "o", "x" }
 ```
 
-### Use Set to ensure the uniqueness of a list of values
+### Use set to ensure the uniqueness of a list of values
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/set/index.md:262]:
```suggestion
### Use a set to ensure the uniqueness of a list of values
```

**Review by Josh-Cena — COMMENTED** [review]
We say "a `Set` object" or "a set".

**Review by Josh-Cena — APPROVED** [review]
Thank you!

**Pull request merged**